server/game/cards/attachments/04/motherofdragons.js

```diff
@@ -0,0 +1,41 @@
+const DrawCard = require('../../../drawcard.js');
+
+class MotherOfDragons extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Kneel Mother of Dragons to add attached character to challenge',
+            method: 'addToChallenge',
+            phase: 'challenge'
+        });
+    }
+
+    addToChallenge(player) {
+        var challenge = this.game.currentChallenge;
+        if(this.kneeled || !player.cardsInPlay.any(card => {
+            return card.inChallenge && card.hasTrait('Dragon');
```

Comment:
Happy to merge as-is, but just so you know, this.game.currentChallenge.isParticipating(card) is a more generic way to do this